From ce530dc934305d271593530066b5349a9f2c3189 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 30 Jan 2013 17:14:56 +0800
Subject: [PATCH 100/231] ENGR00242239-2 Camera app be freezed when taking 5M
 pic in MX6DL

ion heap in MX6DQ only been used for taking picture user case, which
IPU write to the heap buffer, and arm read the buffer for software jpeg encoder.
Make the heap cacheable will increase the jpeg encoder from 2-3s to 0.5-0.7s.

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabresd.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabresd.c b/arch/arm/mach-mx6/board-mx6q_sabresd.c
index 089c52f..eca6966 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabresd.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabresd.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2012 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright (C) 2012-2013 Freescale Semiconductor, Inc. All Rights Reserved.
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -1378,6 +1378,7 @@ static struct ion_platform_data imx_ion_data = {
 		.type = ION_HEAP_TYPE_CARVEOUT,
 		.name = "vpu_ion",
 		.size = SZ_16M,
+		.cacheable = 1,
 		},
 	},
 };
-- 
1.7.10.4

